web/examples/focus-dynamic-interval.html

```diff
@@ -1,195 +1,134 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
     <title>dc.js - Focus Chart with Dynamic Intervals</title>
     <meta charset="UTF-8">
     <link rel="stylesheet" type="text/css" href="../css/bootstrap.min.css">
     <link rel="stylesheet" type="text/css" href="../css/dc.css"/>
-  </head>
-  <body>
+</head>
+<body>
 
-    <div class="container">
-      <script type="text/javascript" src="header.js"></script>
-
-      <p>Zoom test</p>
-      <div id="chart"></div>
-      <div id="range-chart"></div>
+<div class="container">
+    <script type="text/javascript" src="header.js"></script>
 
+    <p>Zoom test</p>
+    <div id="chart"></div>
+    <div id="range-chart"></div>
 
     <script type="text/javascript" src="../js/promise-polyfill.js"></script>
     <script type="text/javascript" src="../js/fetch.umd.js"></script>
-  <script type="text/javascript" src="../js/d3.js"></script>
-  <script type="text/javascript" src="../js/crossfilter.js"></script>
-  <script type="text/javascript" src="../js/dc.js"></script>
-  <script type="text/javascript">
-
-// polyfill Array.find for IE
-// https://tc39.github.io/ecma262/#sec-array.prototype.find
-if (!Array.prototype.find) {
-  Object.defineProperty(Array.prototype, 'find', {
-    value: function(predicate) {
-     // 1. Let O be ? ToObject(this value).
-      if (this == null) {
-        throw new TypeError('"this" is null or not defined');
-      }
-
-      var o = Object(this);
-
-      // 2. Let len be ? ToLength(? Get(O, "length")).
-      var len = o.length >>> 0;
-
-      // 3. If IsCallable(predicate) is false, throw a TypeError exception.
-      if (typeof predicate !== 'function') {
-        throw new TypeError('predicate must be a function');
-      }
-
-      // 4. If thisArg was supplied, let T be thisArg; else let T be undefined.
-      var thisArg = arguments[1];
-
-      // 5. Let k be 0.
-      var k = 0;
-
-      // 6. Repeat, while k < len
-      while (k < len) {
-        // a. Let Pk be ! ToString(k).
-        // b. Let kValue be ? Get(O, Pk).
-        // c. Let testResult be ToBoolean(? Call(predicate, T, « kValue, k, O »)).
-        // d. If testResult is true, return kValue.
-        var kValue = o[k];
-        if (predicate.call(thisArg, kValue, k, o)) {
-          return kValue;
-        }
-        // e. Increase k by 1.
-        k++;
-      }
+    <script type="text/javascript" src="../js/d3.js"></script>
+    <script type="text/javascript" src="../js/crossfilter.js"></script>
+    <script type="text/javascript" src="../js/dc.js"></script>
+    <script type="text/javascript">
+
+        (() => {
```

Comment:
This IIFE was just to get indentation to work... It could be replaced with a simple pair of braces now.